### DISCUSSION TITLE
Support for virtio-iommu

### DISCUSSION THREAD

**Discussion by wesleywesley** [virtio-win/kvm-guest-drivers-windows#987] [Ideas]
Hi, 
We know that in KVM/QEMU/VFIO environment of host + Linux guest
we can use the virtio-iommu front-end in linux guest and virtio-iommu back-end in QEMU (HV).

Just wondering whether there is a similar virtio-iommu frond-end support exist, or plan to be developed?
And if it is, will it work with virtio-iommu back-end in QEMU?

**Comment by YanVugenfirer** [comment]
Hi,

Can you please describe your use case?
Unfortunately, the DDI for implementing the IOMMU driver is not open by Microsoft, so we don't have plans for implementation.

For now you can use emulated Intel or AMD IOMMUSs.

Best regards,
Yan/
